logic.js

```diff
@@ -1,20 +1,20 @@
-const cchangeURL = "https://api.freecurrencyapi.com/v1/latest?apikey=fca_live_FAdHQliAMx0DN6s1YFPRsfsgaoSIUJZiio5rf5aq";
+const cchangeURL = "https://v6.exchangerate-api.com/v6/YOUR_API_KEY/latest";
```

Comment:
this was the old api which was used,i replace with new one because it had few issue in past